plugin.js

```diff
@@ -45,6 +47,162 @@ const globals = new Set([
   '_scrollTo',
 ]);
 
+const blacklistedFunctions = new Set([
```

Comment:
Can you write a comment on how this list of blacklisted was created?

Reply:
Answered in pr description.